src/Store.js

```diff
@@ -201,6 +201,7 @@ class Store {
    * @return {[None]}
    */
   async drop () {
+    await this._cache.open()
```

Comment:
Why open the store on drop?

Reply:
Agreed this prolly shouldn't happen here. Is there a (test) case that requires this to be added? It seems like this is a state management issue that could be managed somewhere else...

Reply:
_Definitely_ shouldn't need to happen here, and yet... 

I believe this happens when calling DB.drop after DB.close, which is what the ODB persistency tests do.

Also if we simply check the status of the DB and don't call open but instead branch on `if cache.status() === "open"` then it never deletes those keys from the cache and we have some dangling data. Also I think there's still a chance the cache will be closed between any of those `del` calls, as in load, so an if open check isn't entirely sufficient.

I'm just not sure where to else to put it because putting it in `Cache.get` is overkill.

Reply:
Removing, will solve elsewhere